### PR TITLE
Update truncate-text-with-ellipsis-in-one-line_en.md

### PR DESCRIPTION
**Pull request by ayechico21**
Solution to the truncate-text-with-ellipsis-in-one-line using overflow property

### CODE REVIEW

File: css/truncate-text-with-ellipsis-in-one-line_en.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
+Solution by [Maneesh Cheema](https://github.com/ayechico21)
 
-There is no solution yet.
+## Using Overflow
 
-Would you like to [contribute to the solution](https://github.com/BFEdev/BFE.dev-solutions/blob/main/css/truncate-text-with-ellipsis-in-one-line_en.md)? [Contribute guideline](https://github.com/BFEdev/BFE.dev-solutions#how-to-contribute)
+```js
+
+.one-line {
+  /* your code here */
```
Comment (BFEdev) [inline css/truncate-text-with-ellipsis-in-one-line_en.md]: please remove this line

Comment (ayechico21) [reply css/truncate-text-with-ellipsis-in-one-line_en.md]: Removed the line

**Comment by ayechico21** [comment]
extra line has been removed @BFEdev

**Comment by BFEdev** [comment]
Thank you

**Pull request merged**